tools/kagami/src/swarm.rs

```diff
@@ -20,13 +20,13 @@ use ui::UserInterface;
 
 use super::{ClapArgs, Outcome};
 
-const GIT_REVISION: &str = env!("VERGEN_GIT_SHA");
-const GIT_ORIGIN: &str = "https://github.com/hyperledger/iroha.git";
+pub const GIT_REVISION: &str = env!("VERGEN_GIT_SHA");
+pub const GIT_ORIGIN: &str = "https://github.com/hyperledger/iroha.git";
```

Comment:
Non-public constants are usually a mistake.

Reply:
I could put the consts into `kagami`'s `mod.rs` after applying this [suggestion](https://github.com/hyperledger/iroha/pull/3610/files#r1242012154) which would be neat.

Reply:
>  It's not necessary to expose everything from a module "just in case".

True. This is not true of constants. 

> I personally think that exposing only essential parts at first place is a better practice. Especially when it is fixable by adding a single pub keyword.

It is true of complex behaviours. The whole point of a constant is that you change it in one place, and the change is propagated.